### PR TITLE
[Proposal] 1: Pool 2

### PR DESCRIPTION
**Pull request by emptysetsquad**
# Proposal 1: Pool 2

#### Changes to the core DAO
- Adds `ESDS` Transfer events when bonding and unbonding.
- Updates LP incentivization pool cut from 5% to 20%.

#### Changes to the LP Incentivization pool
- New LP incentivization pool with:
  - Minor unbonding issue fix
  - Additional `emergencyPause()` and `emergencyWithdraw()` methods

#### Transfer Events
Since `ESDS` is non-transferrable, and in general only partially ERC20-compliant, we did not add in `Transfer` events for bonding and onbonding. Since launch we've noticed that this is having unfavorable effects with blockchain explorers and wallets that rely on these events to update balances. This Proposal will add these ERC20-compliant Transfer events on bond and unbond.

#### Unbonding Fix
We uncovered a minor bug in the LP pool where if a user bonds and unbonds in a specific sequence during a single epoch they may run into an unintended revert, locking their funds until the following epoch advance. No funds were at risk due to this bug, but nonetheless we'd like to fix this while we have a chance. Additionally, we will be adding a `1 ESD` advance reward to the legacy pool temporarily to ensure that no accounts get locked due to this bug during the migration process.

#### Emergency methods
We've added two methods to the new LP pool that are only accessible by the hard-coded DAO address:
- `emergencyPause()` allows the DAO to pause new deposits, bonds, and provides, but does not pause withdrawals, claims, or unbonds.
- `emergencyWithdraw()` allows the DAO to withdraw any ERC20 held by the LP pool with the `to` address again hard-coded as the DAO's.

These methods will remain unused, but will be available as levers in case we need to vote in new DAO logic in the future to respond to an incident with the new pool.

#### Update LP Pool Cut
By setting a fixed LP Pool cut, we can by-proxy set the expected ratio of the total supply supplied as liquidity. Since the protocol is unaudited, and the LP Pool is the only contract that contains non-ESD assets, we wanted to limit the loss potential in the early days of the protocol. Because of this we chose an initial 5% LP cut, purposefully low. Now that we've had time to minimally test the pool, as well as implement the new safety features in this proposal, we are now comfortable raising the pool cut from 5% to 20%, which should theoretically increase liquidity by 4 times.

## Tracking
Implementation: https://etherscan.io/address/0xBe3c2149729a7c001FEfc5b9c1EF829d242dE0CC
Status: Proposal

### CODE REVIEW

File: protocol/test/dao/Bonding.test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const {ZERO_ADDRESS} = require("@openzeppelin/test-helpers/src/constants");
```
Comment (dmaddock1) [inline protocol/test/dao/Bonding.test.js:1]: Pedantic style comment: I notice you space your braces everywhere but here.

**Review by dmaddock1 — COMMENTED** [review]
Looks good.

File: protocol/test/dao/Bonding.test.js (hunk 1 above)
Comment (emptysetsquad) [reply protocol/test/dao/Bonding.test.js]: nice, fixed.

**Comment by Austin-Williams** [comment]
I haven't looked the code, but concept ACK. LGTM.

**Pull request merged**